src/telegram.js

```diff
@@ -1467,6 +1467,19 @@ class TelegramBot extends EventEmitter {
     form.ok = ok;
     return this._request('answerPreCheckoutQuery', { form });
   }
+
+  /**
+   * Use this method to get the sticker set given the name of the set.
+   * Returns a [StickerSet](https://core.telegram.org/bots/api#stickerset) object.
+   *
+   * @param {String} name Name of the sticker set
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
```

Comment:
We are also missing the link to the method, using `@see`. Check out other methods for such links.